integration_tests/src/main/python/asserts.py

```diff
@@ -335,15 +338,24 @@ def assert_gpu_and_cpu_are_equal_collect(func, conf={}):
     In this case the data is collected back to the driver and compared here, so be
     careful about the amount of data returned.
     """
-    _assert_gpu_and_cpu_are_equal(func, True, conf=conf)
+    _assert_gpu_and_cpu_are_equal(func, 'COLLECT', conf=conf)
+
+def assert_gpu_and_cpu_are_equal_count(func, conf={}):
```

Comment:
I am not a huge fan of this API. It makes it really easy to accidentally only compare the number of rows for a query instead of the actual data of the query. I understand why you are trying to do this, but the same thing can be achieved with `assert_gpu_and_cpu_are_equal_collect` and inserting a `groupBy().count()` in there.

This is not a show stopper. If you want to keep it, that is fine, I would just like a warning in the doc string about this.

Reply:
ok, I thought the _count would imply that you are only comparing counts.  We in general have no tests that test count() that I see and we have hit lots of issues with it so I thought it would be a convenient api to easily add tests without having to change the operation or a bunch of code.

I'm definitely open to changing if we think it will cause problems but I don't think groupBy().count() is as intuitive or convenient. 

Perhaps just changing the name to assert_gpu_and_cpu_row_counts_equal and add more documentation?  Or if you can think of another convenience api.

Reply:
That is fine with me. I mostly want to be sure that no one uses it on accident.  Using it on purpose is fine.